src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -257,16 +258,18 @@ export class InstructorCoursesPageComponent implements OnInit {
     const modalRef: NgbModalRef = this.modalService.open(CoursePermanentDeletionConfirmModalComponent);
     modalRef.componentInstance.isDeleteAll = true;
     modalRef.result.then(() => {
-      const paramMap: { [key: string]: string } = {
-        user: this.user,
-      };
-      this.httpRequestService.delete('/instructor/courses/permanentlyDeleteAll', paramMap)
-          .subscribe((resp: MessageOutput) => {
-            this.loadInstructorCourses();
-            this.statusMessageService.showSuccessMessage(resp.message);
-          }, (resp: ErrorMessageOutput) => {
-            this.statusMessageService.showErrorMessage(resp.error.message);
-          });
+      const deleteRequests: Observable<any>[] = [];
```

Comment:
Instead of `Observable<any>[]`, the type can be more specific as `Observable<MessageOutput>[]` to match the return type of `courseService.deleteCourse()`

Reply:
Oh, I followed this [implementation](https://github.com/TEAMMATES/teammates/blob/master/src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts#L577) to use `any`. Thank you very much for your advice